ERCS/erc-7208.md

```diff
@@ -0,0 +1,598 @@
+  * @param mtid The id of ODC to lock.
+  * @param governor The governor of the fractions, if it's empty, msg.sender is used.
+  */
+  function createNewErc721Fractions(
+      string calldata name,
+      string calldata symbol,
+      string calldata baseUri,
+      uint256[] calldata idsToMint,
+      uint256 mtid,
+      address governor
+  ) external returns (address)
+```
+
+## Rationale
+
+The decision to encode Properties as bytes32 data containers in the ODC Interface is primarily driven by flexibility and future-proofing. Encoding as bytes32 allows for a wide range of data types to be stored, including but not limited to strings, integers, addresses, and more complex data structures, providing the developer with flexibility to accommodate diverse use cases. Furthermore, as Ethereum and its standards continue to evolve, encoding as bytes32 ensures that the ODC standard can support future data types or structures without requiring significant changes to the standard itself.
+
+Having a 2-layer data structure of `propertyKey` => `dataKey` => `dataValue` allows different applications to have their own address space. Implementations can manage access to this space using different `propertyKey` for different applications.
+
+A case-by-case example on potential Properties encodings was performed and summarized is provided in the appendix.
+
+The inclusion of Properties within an ODC provides the capability to associate a richer set of on-chain accessible information within the storage. This enables a wide array of complex, dynamic, and interactive use cases to be implemented with multiple ERCs as well as other smart contracts.
+
+Properties in an ODC offer flexibility in storing mutable on-chain data that can be modified as per the requirements of the token's use case. This allows the ODC to hold mutable states and reflect changes over time, providing a dynamic layer to the otherwise static nature of storage through a standardized interface.
+
+By leveraging Properties within the ODC, this standard delivers a powerful framework that amplifies the potential of all ERCs (present and future). In particular, ODCs can be leveraged to represent Account Abstraction contracts, abstracting the data-storage from the logic that consumes it, enabling for a single data-point to have multiple representations depending on the implementation.
+
+
+## Backwards Compatibility
+
+This ERC is intended to augment the functionality of existing token standards without introducing breaking changes. As such, it does not present any backwards compatibility issues. Already deployed ERCs can be wrapped as Properties, with the application of on-chain data relevant to each use-case.
+
+It offers an extension that allows for the storage and retrieval of Properties within an ODC while maintaining compatibility with existing ERCs related to tokenization.
+
+
+## Reference Implementation
+
+See Nexera Protocol (nexeraprotocol dot com)
```

Comment:
Please remove this external link. If you'd like to include a reference implementation, you may do so in the assets directory under a permissive license (preferably CC0-1.0).